__tests__/app.test.js

```diff
@@ -47,4 +47,40 @@ describe("GET /api/", () => {
         });
 
     });
+    describe("/articles/:article_id",() => {
+        test("200: status code and responds with object corresponding to article_id",async() => {
+            const { status, body } = await request(app).get("/api/article/1")
+            const {article} = body
+console.log(article)
+            const expectedArticle =  {
+                article_id:expect.any(Number),
```

Comment:
I would expect article_id to be 1 here - proving you have the correct article. You could also be more specific with the other properties but I would do article_id as a minimum. We should know what it is